xcm/transfers_dev.json

```diff
@@ -35,6 +35,32 @@
     "UNIT": {
       "chainId": "e1ea3ab1d46ba8f4898b6b4b9c54ffc05282d299f89e84bd0fd08067758c9443",
       "multiLocation": {}
+    },
+    "ACA": {
+      "chainId": "fc41b9bd8ef8fe53d58c7ea67c794c7ec9a73daf05e6d54b14ff6342c99ba64c",
+      "multiLocation": {
+        "parachainId": 2000,
+        "generalKey": "0x0000"
+      }
+    },
+    "DOT": {
+      "chainId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",
+      "multiLocation": {}
+    },
+    "GLMR": {
+      "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+      "multiLocation": {
+        "parachainId": 2004,
+        "palletInstance": 10
+      }
+    },
+    "RMRK": {
+      "chainId": "48239ef607d7928874027a43a67689209727dfb3d3dc5e5b03a39bdc2eda771a",
+      "multiLocation": {
+        "parachainId": 1000,
+        "palletInstance": 50,
+        "generalKey": "8"
```

Comment:
We need to check this. Probabaly tokens from Assets pallet are referenced by general index

Reply:
fixed, is this correct?

Reply:
done